client/src/components/Login.js

```diff
@@ -3,39 +3,56 @@ import { Link, useNavigate } from "react-router-dom";
 import axios from 'axios';
 import { useUser } from "../contexts/UserProvider";
 import { BiArrowBack } from "react-icons/bi";
+import Alert from 'react-bootstrap/Alert';
+
 
 function Login() {
 	const [credentials, setCredentials] = useState({ email: "", password: "" });
-	const navigate = useNavigate();
+	const [show, setShow] = useState(false);
+	const navigate = useNavigate(); 
 	const { setUser } = useUser();
 
 	const handleSubmit = async (e) => {
 		e.preventDefault();
 
 		try {
-			const response = await axios.post(process.env.REACT_APP_SERVER_URL + "auth", credentials)
-			const { success, token, user } = response.data;
-
-			if (!success) return alert("Invalid credentials!");
+			const response = await axios.post(`${process.env.REACT_APP_SERVER_URL}auth`, {
+				"email": credentials.email,
+				"password" : credentials.password
+			})
+			const {token, user } = response.data;
```

Comment:
destructure `success` too and if it's not a success then also show the alert.

Reply:
![image](https://user-images.githubusercontent.com/73685420/227702223-c1d0b18f-8bed-481f-9bbc-098f644fe4cb.png)
will that do?